packages/api-reference/src/components/Content/Lazy/Loading.vue

```diff
@@ -114,7 +114,7 @@ watch(
     }
     // Descriptions
     else {
-      scrollToId(hash.value)
+      if (typeof window !== 'undefined') scrollToId(hash.value)
```

Comment:
Should we put this in scrollToId? Because there’s no cse where window if undefined and scrollToId will work.

Reply:
yea I thought about that, but this is nice for catching errors

Reply:
we can do either way!